### ISSUE TITLE
Fix parsing errors detected in the app to meet a11y standards

### ISSUE THREAD

**Issue by ajpeddakotla** [hypothesis/product-backlog#787]
This issue is part of the larger accessibility epic written up here #753

Problem:
In content implemented using markup languages, elements must have complete start and end
tags, be nested according to their specifications, not contain duplicate attributes, and have
unique IDs, except where the specifications allow these features. These types of errors can
interfere with the operation of assistive technologies. For more information on this see W3C page here: https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html

Parsing Errors currently detected (93 errors and warnings), including:
• missing required attributes
• invalid nesting of elements

In general, ensure valid code by checking pages with the W3C Nu Validator
(https://validator.w3.org/nu/). The raw results show all of the source code validity issues, but to
Pass this criterion only a sub-set of these issues must be fixed. This subset can be filtered using a special “bookmarklet” found at: http://www.paciellogroup.com/blog/2012/02/wcag-2-0-
parsing-error-bookmarklet/

Acceptance Criteria: 
- [ ] Determine the set of parsing errors need to address in order to meet a11y guidelines (per Nu Validator and bookmarklet)
- [ ] Fix those errors
- [ ] Run the app through the validator and bookmarklet again to ensure we've corrected the problem.

**Comment by ajpeddakotla** [comment]
| User Engagement | Building Community | Level of Effort |
|---|---|---|
| 2 | 1 | 3 |

**Comment by ajpeddakotla** [comment]
Closed in favor of https://github.com/hypothesis/product-backlog/issues/1085
